src/generate.ts

```diff
@@ -254,20 +322,24 @@ export default async (options: GeneratorOptions) => {
 
         const tmpDir = fs.mkdtempSync(os.tmpdir() + path.sep + 'prisma-erd-');
 
-        // https://github.com/notiz-dev/prisma-dbml-generator
+        // https://github.com/notiz-dev/prisma-dbml-generator # TODO What does this comment signify?
```

Comment:
I believe I added that comment when I was trying to learn how to make a Prisma generator. It's unnecessary now.